### PR TITLE
Update Russian localization

### PR DESCRIPTION
**Pull request by Whiteychannel**
Performed a Russian translation for version 1.5.0 except for "Deafen members" and "View guild insights"

### CODE REVIEW

File: lang/ru.json [hunk 1]
```diff
@@ -22,7 +25,106 @@
...
+        },
+        "off": {
+          "title": "Логи выключены",
+          "description": "Функция ведения логов отключена."
+        }
+      }
+    },
+    "merge": {
+      "title": "Объединить голосовой канал",
+      "description": "Переместите участников голосового канала, к которому вы подключены, в другой. Эта команда может быть выполнена только в том случае, если звонящий и бот имеют разрешение на перемещение участников.",
```
Comment (Walkyst) [inline lang/ru.json:107]:
```suggestion
      "description": "Переместите участников голосового канала, к которому вы подключены, в другой. Эта команда может быть выполнена только в том случае, если пользователь запросивший команду и бот имеют разрешение на перемещение участников.",
```

File: lang/ru.json [hunk 2]
```diff
@@ -38,7 +140,54 @@
...
+    "PRIORITY_SPEAKER": "Приоритетный режим",
+    "STREAM": "Возможность вести стрим",
+    "VIEW_CHANNEL": "Чтение канала",
+    "SEND_MESSAGES": "Отправка сообщений",
+    "SEND_TTS_MESSAGES": "Отправка TTS сообщений",
+    "MANAGE_MESSAGES": "Управление сообщениями",
+    "EMBED_LINKS": "Просмотр ссылок",
+    "ATTACH_FILES": "Прикрепить файлы",
+    "READ_MESSAGE_HISTORY": "Доступ к истории каналов",
+    "MENTION_EVERYONE": "Упомянание (@)everyone",
```
Comment (Walkyst) [inline lang/ru.json:180]:
```suggestion
    "MENTION_EVERYONE": "Упоминание (@)everyone",
```

**Comment by koloml** [comment]
@Whiteychannel apply suggestions from @Walkyst.

**Comment by koloml** [comment]
Everything looks ok.

**Pull request merged**